cra-config.yaml

```diff
@@ -1,11 +1,6 @@
 # More info about this file at https://github.com/terraform-ibm-modules/common-pipeline-assets/blob/main/.github/workflows/terraform-test-pipeline.md#cra-config-yaml
 version: "v1"
 CRA_TARGETS:
-  - CRA_TARGET: "examples/basic" # Target directory for CRA scan. If not provided, the CRA Scan will not be run.
+  - CRA_TARGET: "modules/access-management" # Target directory for CRA scan. If not provided, the CRA Scan will not be run.
```

Comment:
CRA needs to be run against an example / solution that has a provider.tf in it. Its not supposed to run against a module

Reply:
updated to `examples/access-management`